migrate-dao.js

```diff
@@ -133,8 +130,7 @@ async function migrateDAO ({ arcVersion, web3, spinner, confirm, opts, migration
         '0'
       )
 
-      tx = await forgeOrg.send({ nonce: ++nonce })
-
+      tx = (await sendTx(forgeOrg), 'Creating a new organization...').receipt
```

Comment:
Misplaced `)`, this should be `(await sendTx(forgeOrg, 'Creating a new organization...')).receipt`